Automattic/pocket-casts-android
# Custom Playback Settings: Add Feature Flag and add segmented tab bar

## Description
This adds the `custom_playback_settings` Feature Flag and the segmented tab bar in the playback effects view.

** Ignore local files for now

Project #3042

## Testing Instructions

1. Play an episode and tap effects icon in full-screen player
2. ✅ Notice that the segmented tab bar is displayed at the top of the effects view** (hHFpI1RtnW4TRb448Q4Don-fi-4730_28640)
3. Disable the FF
4. Re-open the effects view and
5. ✅ Notice that the segmented tab bar is not displayed 
6. Go to Podcasts tab
7. Tap on locked folder icon
8. ✅ Notice that segmented bar on the Plans page looks fine

** Only match the segmented control in Figma, the top header is highlighted only on iOS.

## Screenshots or Screencast 

**Effects Settings**
<img width=320 src="https://github.com/user-attachments/assets/bd2df157-5cc0-4ae9-a8e5-9f2d702b2dda"/>

**Plans**
<img width=320 src="https://github.com/user-attachments/assets/408d9606-9d97-49df-8b7a-5b09fe9f567d"/>


## Checklist
 
#### I have tested any UI changes...
